allensdk/brain_observatory/ecephys/write_nwb/__main__.py

```diff
@@ -157,6 +157,13 @@ def read_spike_amplitudes_to_dictionary(
 
     templates = load_and_squeeze_npy(templates_path)
     spike_templates = load_and_squeeze_npy(spike_templates_path)
+    inverse_whitening_matrix = load_and_squeeze_npy(inverse_whitening_matrix_path)
+
+    for temp_idx in range(templates.shape[0]):
+        templates[temp_idx,:,:] = np.dot(
```

Comment:
There are templates and  spike_templates. The original issue referred spike templates, but here you are modifying templates. It would help having a docstring or a comment describing how the two are related.

Reply:
these are documented here:
https://github.com/AllenInstitute/AllenSDK/blob/64f18e62af93bd88f469a97ee3825236dc28f13d/allensdk/brain_observatory/ecephys/write_nwb/_schemas.py#L90

briefly, spike templates is a look up table from spikes to the templates used to extract them, templates are the actual template waveforms.